mantidproject/mantid
# Cppcheck conda environment setup

**Description of work.**
Adds a script for setting up a conda environment, and activating it, before building the cppcheck target. This is necessary for the new `isis-cloud-linux` machines which do not have cmake and ninja, etc pre-installed.

**To test:**
Code review and check the following jobs 

See jobs on https://builds.mantidproject.org/job/pull_requests-cppcheck/

- Job 53779 - a cppcheck failure and a generated cppcheck.xml artifact
- Job 53777 - skipping cppcheck because no c++ changes were made





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
